__test__/ResponsiveCardsRow.test.js

```diff
@@ -15,9 +16,19 @@ describe(`ResponsiveCardsRow`, () => {
         batmanFilmsSpeciesCardProps,
         findingNemoSpeciesCardProps,
         theSmithHouseholdImageCardProps
-      ]
+      ],
+    CardClass: () => `topClass`
   }
 
+  test(`displays cards in default row mode`, () => {
+    const wrapper = shallow(<ResponsiveCardsRow {...props}/>)
+    expect(wrapper.find(Slide)).toHaveLength(0)
+  })
```

Comment:
Actually, `ResponsiveCardsRow` doesn’t know what a `Slider` is. You won’t see it anywhere in its source code, so why should it care? You can remove this test and be all the happier. :smile_cat:

Reply:
Yes. This test has been removed and test coverage is 100% by adding one more snapshots with different `slideSettings` for `CarouselCardsRow`.